tencentcloud/resource_tc_kubernetes_node_pool.go

```diff
@@ -910,8 +930,21 @@ func composeAsLaunchConfigModifyRequest(d *schema.ResourceData, launchConfigId s
 		request.InternetAccessible.PublicIpAssigned = &publicIpAssigned
 	}
 
-	if v, ok := dMap["security_group_ids"]; ok {
-		request.SecurityGroupIds = helper.InterfacesStringsPoint(v.(*schema.Set).List())
+	if d.HasChange("auto_scaling_config.0.security_group_ids") {
+		if v, ok := dMap["security_group_ids"]; ok {
+			if list := v.(*schema.Set).List(); len(list) > 0 {
+				errRet := fmt.Errorf("Parameter `security_group_ids` has been deprecated and cannot be changed anymore, please use `orderly_security_group_ids` instead it.")
```

Comment:
不支持的原因能否说清楚点：
因为它不能保证安全组的先后顺序，会给您的业务带来风险。

Reply:
改成这样子？ok不
The parameter "security_group_ids" is deprecated because the order of elements in this field cannot be guaranteed, therefore adding security groups is no longer supported. Please use "orderly_security_group_ids" instead.

The parameter "security_group_ids" is deprecated because the order of elements in this field cannot be guaranteed, therefore changing security groups is no longer supported. Please use "orderly_security_group_ids" instead.